frontend-cert/react-projects/react-use-effect/user-stories.md

```diff
@@ -0,0 +1,12 @@
+Your `"Generate New OTP"` button should display a new 6-digit OTP in the `h2` element when clicked.
```

Comment:
We will need to add a short description here because a lot of people won't know what OTP means

Reply:
I have defined it in the description part.